include/kernel.h

```diff
@@ -5037,21 +5037,21 @@ struct k_poll_event {
 	};
 };
 
-#define K_POLL_EVENT_INITIALIZER(event_type, event_mode, event_obj) \
+#define K_POLL_EVENT_INITIALIZER(type_event, event_mode, event_obj) \
 	{ \
 	.poller = NULL, \
-	.type = event_type, \
+	.type = type_event, \
```

Comment:
reversing the string here give it a completely new meaning, giving names is often very difficult, but changing them to something that changes the meaning or make it less obvious is worse than the issue you are trying to fix.

Reply:
for macro argument names I prefer to just prefix all of them with `_`.

so for example:

`#define K_POLL_EVENT_INITIALIZER(_event_type, _event_mode, _event_obj)`

Reply:
Added the changes.